packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -435,9 +436,24 @@ export interface IContainerRuntimeOptions {
 	 * By default, the feature is disabled. If enabled from options, the `Fluid.ContainerRuntime.DisableGroupedBatching`
```

Comment:
I'm not sure why we need to combine MergeTree changes with ContainerRuntime changes.
I'd hope that MergeTree can submit their changes (with individual UTs) independently, no?

Reply:
The reason Tony's changes have been merged into this PR is to showcase the whole thing working end to end. This will be merged after the MergeTree changes make it into main, on their own.